www/templates/default/html/manager/Controller.tpl.php

```diff
@@ -41,7 +41,7 @@
 $page->addStyleDeclaration("#dcf-mobile-toggle-menu {display: none!important}");
 
 //javascript
-$page->addScriptDeclaration('WDN.setPluginParam("idm", "logout", "' .  Util::getBaseURL() . '/manager/logout");');
+$page->addScriptDeclaration('WDN.setPluginParam("idm", "logout", "logout");');
```

Comment:
Still does https://events-dev.unl.edu/manager/TractorSafetyTraining/logout on the manager side.  Probably still need to prepend a working base url string